citest/json_predicate/path_result.py

```diff
@@ -181,6 +181,28 @@ def __init__(self, source, target_path, path_value=None, **kwargs):
         path_value=path_value, **kwargs)
 
 
+class UnexpectedPathError(PathResult):
+  """A PathResult indicating an object contained an unexpected attribute."""
+
+  def __init__(self, source, target_path, path_value, **kwargs):
+    """Constructor.
+
+    Args:
+      source: [obj] The original JSON object path_value is relative to.
```

Comment:
`obj` or `dict`? I think by default JSON objects are represented as `dict`s in python.